__tests__/integration/nanocontracts/upgrade_contract.test.ts

```
@@ -0,0 +1,121 @@
+    const txInit = await hWallet.createAndSendNanoContractTransaction(
+      NANO_CONTRACTS_INITIALIZE_METHOD,
+      address0,
+      { blueprintId: v1, args: [] }
+    );
+    await checkTxValid(hWallet, txInit);
+    const ncId = txInit.hash;
+
+    // V1's `set_value` accepts an Address — a wallet-owned base58 string works.
+    const txSetV1 = await hWallet.createAndSendNanoContractTransaction(
+      'set_value',
+      address0,
```

ESLint

[error] Replace `⏎······'set_value',⏎······address0,` with `'set_value',·address0,·{`

(prettier/prettier)